SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -1497,39 +1480,39 @@ void CFlowOutput::WriteForcesBreakdown(const CConfig* config, const CSolver* flo
         break;
 
       case INC_IDEAL_GAS:
-        file << "Fluid Model: INC_IDEAL_GAS " << endl;
-        file << "Variable density incompressible flow using ideal gas law." << endl;
-        file << "Density is a function of temperature (constant thermodynamic pressure)." << endl;
-        file << "Specific heat at constant pressure (Cp): " << config->GetSpecific_Heat_Cp() << " N.m/kg.K." << endl;
-        file << "Molecular weight : " << config->GetMolecular_Weight() << " g/mol" << endl;
-        file << "Specific gas constant: " << config->GetGas_Constant() << " N.m/kg.K." << endl;
+        file << "Fluid Model: INC_IDEAL_GAS \n";
```

Comment:
I once saw a video from Mr Turner that using endl to line break is much slower compared to \n. Was that you primary motivation or are there other considerations to this?

Ok it was actually so important to him that it was [Ep 7 of c++ weekly](https://www.youtube.com/watch?v=GMqQOEZYVJQ) back in the good'ol days

Reply:
Yep that as well, and also clang-format looked nicer.